validation/helpers.py

```diff
@@ -206,9 +206,45 @@ def get_distance_with_translations(word):
         matches = get_translations(results)
 
         suggestions[word] = {
+            "transcription": word,
             "med": suggestions[word],
             "matches": matches,
             "len": len(matches) if len(matches) == 1 else len(matches) + 1,
         }
 
     return suggestions
+
+
+def perfect_match(word, suggestions):
+    # Checks for exactly one entry with either
+    # MED = 0 or an exact match for spelling
+    # Returns none if no matches found, or if multiple matches found
+
+    match = None
+    for suggestion in suggestions:
+        if suggestions[suggestion]["med"] == 0 or suggestion == word:
+            if match is None:
+                match = suggestions[suggestion]
+            else:
+                return None
```

Comment:
Ah, so this is in the case of a duplicate?

Reply:
Yes. It's not the most elegant way of doing it, but it does fail fast in the case of failure.